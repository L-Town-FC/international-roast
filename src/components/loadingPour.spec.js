import React from "react"
import renderer from "react-test-renderer"

import LoadingPour from "./loadingPour"

describe("Header", () => {
  it("renders correctly", () => {
    const tree = renderer
      .create(<LoadingPour />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})