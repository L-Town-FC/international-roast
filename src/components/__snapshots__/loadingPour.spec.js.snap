// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header renders correctly 1`] = `
<div
  style={
    Object {
      "display": "flex",
      "justifyContent": "center",
    }
  }
>
  <img
    alt="pouring"
    src="test-file-stub"
    style={
      Object {
        "height": "40vh",
      }
    }
  />
</div>
`;
